src/components/StepButtons.js

```diff
@@ -85,7 +85,7 @@ const StepButtons = ({ tabs, activeTab, onChangeTab, finalStep }) => {
       ([i, { key, title, isValid }]) => stepButton({ i: i * 1, key, title, isValid, selectedIndex, onChangeTab, tabs }),
       _.toPairs(tabs)
     ),
-    finalStep && _.every('isValid', tabs) && h(Fragment, [
+    finalStep /*&& _.every('isValid', tabs)*/ && h(Fragment, [
```

Comment:
Is there a specific reason to leave the commented section in?

Reply:
I forgot to remove it. I was in the middle of making sure it wasn't needed anymore. I'm glad it's still there, though, because it reminds me that I didn't finish thinking through it.

Reply:
Confirming change. Intent is to show a disabled button instead of conditionally showing the button. Better to show the user what they could do even if the action is disabled for some reason.